.travis.yml

```diff
@@ -8,26 +8,32 @@ php:
   - 7.0
   - hhvm
 
-matrix:
-  allow_failures:
-    - php: 7.0
-    - php: hhvm
+sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
```

Comment:
> All dependencies will be updated if necessary. 

Didn't find the information on how this actually works in Travis doc. 

> Which one? And BTW, Makefile is not used on Travis AFAIK.

Travis isn't concerned here, but contributors/hackers are. Please read the `README.md` to understand what the `Makefile` stands for. Your changes on `bin` dir actually makes the `Makefile` bugging.